### PR TITLE
Adds loads of new berets

### PR DESCRIPTION
**Pull request by Xantholne**
:cl: Xantholne
add: Loads of new berets are now available in respective lockers for Captain, Blueshield, CMO, Medical, Science, Chief Engineer, Atmos tech, Station engineer, Warden and some new CC berets.
/:cl:

These have been requested frequently, as people love berets. As such this adds in a whole bunch of new berets some I made others ported from Para.

First column on the left is Warden, Warden Corporate (Not added to locker currently), Security Officer Corporate (Not added to locker currently), Chief Engineer, Atmos tech, Station Engineer, Scientist (Also available to RD and Robo), CMO (Originally modeled after the paracode CMO jacket, possibly will have its' colors altered to the more darker tg code jacket), Medical Doctor, Captain, CC Captain, CC Officer (With a blueshield variant in their locker), CC Navy (Also with a blueshield variant in their locker).

As noted, it's likely I'll tone the colors on the CMO beret down, since it does not entirely match the color scheme with the CMO jacket on here unlike it did from paradise.

The corporate Warden and Officer berets are currently not in lockers but based on feedback I may add them to their respective lockers (With only one or two at most for the security officer).

All berets have very minor stats aside from the Security, Blueshield, and Captain ones to make them competitive against other hats so style is not sacrificed for power.

beret preview

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Instamerge, I need this

**Comment by AdamElTablawy** [comment]
engineer beret too shiny

**Comment by Ziiro** [comment]
>engineer beret too shiny
It's insulated.

**Comment by Ziiro** [comment]
ALSO REEEEEEE VOX SPRITES

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -166,3 +187,89 @@
...
+	strip_delay = 60
+
+/obj/item/clothing/head/beret/blueshieldnavy
+	name = "navy officer beret"
+	desc = "A navy Blueshield beret."
+	icon_state = "beret_centcom_officer_navy"
+	armor = list(melee = 40, bullet = 20, laser = 10, energy = 10, bomb = 10, bio = 5, rad = 5, fire = 5, acid = 30)
+	strip_delay = 60
+
+/obj/item/clothing/head/beret/Captain
```
Comment (FlattestGuitar) [inline code/modules/clothing/head/jobs.dm]: Uncapitalize this and we can merge.

**Comment by FlattestGuitar** [comment]
The CMO jacket has to be recolored, not the other way around. The turquoise puké shade is horrid.

**Comment by Xantholne** [comment]
I'll recolor it to match the beret then in either the next or second to next PR that does some uniforms stuff.

**Pull request closed without merging**